From 2a30f4ccfa2791789831863224323037889bb064 Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Fri, 26 Oct 2018 10:23:25 +0800
Subject: [PATCH 2268/2278] isys & psys package lib2600b0 for commit-id aa46345

FW_Manifest_ID: ea3e0f2355794b6a8475680196d2c634dabac0de

FW_Build_Branch: stable_iot_mr4_pv_r1_20180817

FW_Build_Topic: auto_merge_queue

library version: commit-id aa46345

FW Bundle patch number of this release :  364157

FW tag : scci_stable_iot_mr4_pv_r1_20180817_20181024_1305

Change-Id: Ie47ab4e74c90daf15b5a9d11b4af59e3d24acc26
Depends-on: I0e22350ed93571345c01e507d3ecaf2c900e3b11
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h           | 2 +-
 .../lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index ec3fbd8..fd7dc00 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180925
+#define IA_CSS_FW_PKG_RELEASE  0x20181024
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index ec3fbd8..fd7dc00 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20180925
+#define IA_CSS_FW_PKG_RELEASE  0x20181024
-- 
2.7.4

